dspace-api/src/test/java/org/dspace/eperson/GroupTest.java

```diff
@@ -206,6 +206,9 @@ public void findAllNameSort() throws SQLException {
         List<String> names = new ArrayList<>();
         List<String> sortedNames = new ArrayList<>();
         for (Group group : groups) {
+            if (group.getName() == null) {
+                continue;
```

Comment:
If I run the existing set of tests against postgres, there are dozens of groups, epersons, items, and collections that persist.  I am finding it particularly challenging to delete EPersons and Groups in the new test that I am writing.  For the new tests, I am assuming that I will need to wipe out the database between runs.  If we want the tests to clean up after themselves, that should probably be captured in a new (more complex) ticket.